Net/src/HTTPStream.cpp

```diff
@@ -43,7 +43,8 @@ void HTTPStreamBuf::close()
 	if (_mode & std::ios::out)
 	{
 		sync();
-		_session.socket().shutdownSend();
+		if (!_session.getKeepAlive())
```

Comment:
This is unnecessary as HTTPOutputStream is not used with persistent connections.